ui/src/store/modules/oidc.ts

```diff
@@ -0,0 +1,11 @@
+import { vuexOidcCreateStoreModule } from 'vuex-oidc'
+
+export const oidc = vuexOidcCreateStoreModule({
+  // authority: process.env.AUTH_AUTHORITY,
+  // client_id: process.env.AUTH_CLIENT_ID,
+  authority: 'https://auth.lindas.admin.ch/realms/ref',
+  clientId: 'datacube-zazukoians',
+  redirectUri: window.location.origin + '/oidc-callback',
+  responseType: 'code',
+  scope: 'profile',
```

Comment:
there are now `pipelines:read` and `pipelines:write` scopes with which you can experiment with